docs/migrating.md

```diff
@@ -2,12 +2,24 @@
 
 ## CSS changes
 
-While Warp maintains as much backward-compatibility as possible,
+While Warp maintains as much backward-compatibility as possible from Fabric,
 some breaking changes were necessary or justified.
 
+### How to migrate
+1. Make sure to install warp to your project. Follow the instructions on https://warp-ds.github.io/tech-docs/getting-started/
+2. Remove Fabric as a dependency from your project
+3. Remove any mentions of Fabric, e.g. `fabric.css`
+4. Change the prefixes `f-` to `w-` for your component names. The icons should still use the `f-` prefix. Use the `migration plugin` to help with this (see further down this page for further instructions on using the plugin)
```

Comment:
I think you can add a hyperlink to `migration plugin` so that it points to https://warp-ds.github.io/tech-docs/getting-started/#migration-plugin

Reply:
Yeah, very nice!